src/cache/precomputed_quantities.jl

```diff
@@ -366,7 +383,59 @@ function set_precomputed_quantities!(Y, p, t)
                     ᶜtsʲ,
                     ᶜspecificʲ.e_tot,
                 )
+
+                # Overwriting the first interior point for EDMFX ᶜh_totʲ
+                # based on boundary conditions for updrafts
+                ᶜh_tot_int = Fields.level(ᶜh_tot, 1)
+                ᶜh_totʲ_int = Fields.level(ᶜh_totʲ, 1)
+                @. ᶜh_totʲ_int = sgs_h_tot_first_interior_bc(
```

Comment:
I think you may need to do something like:
```
ᶜh_totʲ_int_value = Fields.field_values(ᶜh_totʲ_int)
@. ᶜh_totʲ_int_value = sgs_h_tot_first_interior_bc(...)
```
I'm surprised it doesn't work for `Field` though.